pkgs/tools/networking/dsniff/default.nix

```
@@ -49,7 +49,7 @@ in stdenv.mkDerivation {
     domain = "salsa.debian.org";
     owner = "pkg-security-team";
     repo = "dsniff";
-    rev = "debian%2F2.4b1%2Bdebian-29"; # %2B = urlquote("+"), %2F = urlquote("/")
+    rev = "debian/2.4b1+debian-29"; # %2B = urlquote("+"), %2F = urlquote("/")
```

The comment is no longer needed.
We can use macro for the version.
```suggestion
    rev = "debian/$(version)+debian-29";
```

:warning: This change requires to add ` rec` after `stdenv.mkDerivation`